pkg/kdump/kdump-view.jsx

```diff
@@ -41,14 +41,70 @@ import { Tooltip, TooltipPosition } from "@patternfly/react-core/dist/esm/compon
 import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
 
 import { useDialogs, DialogsContext } from "dialogs.jsx";
+import { read_os_release } from "os-release.js";
 import { fmt_to_fragments } from 'utils.jsx';
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 import { FormHelper } from "cockpit-components-form-helper";
 import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { ModificationsExportDialog } from "cockpit-components-modifications.jsx";
 
 const _ = cockpit.gettext;
 
+const exportAnsibleTask = (settings, os_release) => {
+    const target = Object.keys(settings.targets)[0];
+    const targetSettings = settings.targets[target];
+    const kdump_core_collector = settings.core_collector;
+
+    let role_name = "rhel-system-roles";
+    if (os_release.NAME.includes('Fedora')) {
+        role_name = "linux-system-roles";
+    }
+
+    let ansible = `
+---
+# ${role_name} is also available via ansible-galaxy https://galaxy.ansible.com/ui/standalone/roles/linux-system-roles/kdump/
+- name: install ${role_name}
+  package:
+    name: ${role_name}
+    state: present
+  delegate_to: 127.0.0.1
+  become: true
+- name: run kdump system role
+  include_role:
+    name: ${role_name}.kdump
+  vars:`;
+
+    if (target === "local") {
+        ansible += `
+    kdump_path: ${targetSettings.path || "/var/crash"}
```

Comment:
Still to be done

Reply:
Sent https://github.com/cockpit-project/cockpit/pull/19633 but I've also now moved things around as every condition defined kdump_path and kdump_core_collector so that is now less duplicated.